website/source/api/acl/acl.html.md

```diff
@@ -263,3 +263,112 @@ agent_prefix "" {
    policy = "read"
 }
 ```
+
+## Login to Auth Method
+
+This endpoint was added in Consul 1.5.0 and is used to exchange an [auth
+method](/docs/acl/acl-auth-methods.html) bearer tokens for a newly-created
```

Comment:
```suggestion
method](/docs/acl/acl-auth-methods.html) bearer token for a newly-created
```
Assuming you didn't mean this to be plural. If its meant to plural, remove the "an"  from the previous line.